### ISSUE TITLE
"Object is possibly null" even inside v-if check.

### ISSUE THREAD

**Issue by subdavis** [Yuyz0112/vue-type-check#11]
```
<div v-if="foo !== null">
    {{ foo.content }}
</div>
```

```
274:11 Object is possibly 'null'.
  272 |       <div>
> 274 |         {{ foo.content }}
      |            ^^^^^^^
  275 |       </div>
  276 |     </template>
```

Vetur doesn't complain about this.  I don't really know what to do about it.

**Comment by subdavis** [comment]
Appears to be an upstream bug: https://github.com/vuejs/vetur/issues/1635#issuecomment-658220042

**Comment by dominikjasek** [comment]
Hi @subdavis, did you manage to fix this? I am having the same problem
